bin/dart_curso04_exceptions.dart

```diff
@@ -1,25 +1,59 @@
-import 'controllers/bank_controller.dart';
-import 'models/account.dart';
+// void main() {
+//   // Criando o banco
+//   BankController bankController = BankController();
+//
+//   // Adicionando contas
+//   bankController.addAccount(
+//       id: "Ricarth",
+//       account:
+//           Account(name: "Ricarth Lima", balance: 400, isAuthenticated: true));
+//
+//   bankController.addAccount(
+//       id: "Kako",
+//       account:
+//           Account(name: "Caio Couto", balance: 600, isAuthenticated: true));
+//
+//   // Fazendo transferência
+//   bool result = bankController.makeTransfer(
+//       idSender: "Kako", idReceiver: "Ricarth", amount: 700);
+//
+//   // Observando resultado
+//   print(result);
+// }
 
 void main() {
-  // Criando o banco
-  BankController bankController = BankController();
+  int i = 0;
+  print("Started Main");
+  function01();
```

Comment:
sempre tente usar uma função com mais significado. Se o objetivo é testar exception ou o que for, espresse isso no nome da função.